### DISCUSSION TITLE
Support Needed:  The HTTP Server Must Have Write Permissions

### DISCUSSION THREAD

**Discussion by Steeltooth493** [FreshRSS/FreshRSS#6571] [Q&A] [answered]
I installed and started the Freshrss docker container in Container Manager on my Synology NAS using the latest official image available in Container Manager.  I went through the Container Manager GUI and successfully started the container.  When I go to the local IP address for Freshrss I get to the Web GUI and it gives me an HTTP server write permissions error in the file installation section, so I cannot proceed further.  I'm not very familiar with how to access Docker Compose from the Container Manager GUI to update it that way.  How can I fix this error? 
![FreshRSS Error Container Manager Config 1](https://github.com/FreshRSS/FreshRSS/assets/50289117/e1eaa74b-189b-4077-bf73-3a355594675f)
![FreshRSS erroor 1](https://github.com/FreshRSS/FreshRSS/assets/50289117/5e82ee5b-52af-4bd1-a38e-13c84b489bfe)

**Comment by Alkarex** [comment]
This is probably due to some manual volumes (with wrong permissions) instead of using Docker volumes in your Docker Compose (which handle permissions automatically)

**Reply by Steeltooth493** [reply]
Then to fix this should I check my Docker permissions for the manual freshrss data volume folder that I have created?

**Reply by Alkarex** [reply, marked as answer]
Indeed, and you can also check from inside the container, with `docker exec freshrss ls -alh /freshrss_data`
Furthermore, if you keep the default data path (which is `/var/www/FreshRSS/data`) instead of overriding with a custom path with `DATA_PATH`, the Docker container will automatically try to ensure the correct rights upon restart.

**Reply by Steeltooth493** [reply]
Got it fixed!  I stopped the container, updated the Environment data_path field from /freshrss_data to the default path at /var/www/FreshRSS/data and restarted the container.  Then it worked!  Thank you!
